### ISSUE TITLE
Narrator loses focus on the display menu options and reads what's behind it

### ISSUE THREAD

**Issue by lrabinowicz** [OfficeDev/office-ui-fabric-react#11838]
### Environment Information

- **Package version(s)**: 7.86.0
- **Browser and OS versions**: Microsoft Edge, Windows 10

### Describe the issue:

When drop down menu is open, narrator is reading what is behind the menu options. Also, when going to the last menu option and pressing arrow down, focus goes out of the menu options.

### Please provide a reproduction of the issue in a codepen:

https://codepen.io/LuciaRabinowicz/pen/povXNYx?editors=1010

Repro steps:
1- Turn narrator on
2- Open the display menu 
3- Click up arrow until you are in the first menu option
4-  Click up arrow one more time. 
5 - Focus is now behind the menu options.

1- Turn narrator on
2- Open the display menu 
3- Click down arrow until you are in the last menu option
4-  Click down arrow one more time. 
5 - Focus is not on the menu options any more.

Open the codepen in Microsoft Chromium Edge and follow the same steps, the behavior here is the expected one. 

#### Expected behavior:

If the display menu is open, focus should not be on the elements that are behind it. It should be just in the display menu options until is closed. 

### Documentation describing expected behavior
 Same as in Microsoft Chromium Edge

**Comment by khmakoto** [comment]
Hi @lrabinowicz, thanks for filing this issue! I think you might be seeing this behavior because you have scan mode turned on, as I can also repro it that way, but if I turn scan mode off then the correct behavior is happening. Since scan mode breaks author's intent regarding focus I think there's not much we can do here and this is by design with Narrator+Edge. Can you confirm if this happens even with scan mode turned off?

**Comment by lrabinowicz** [comment]
Is just happening with the scan mode on. Thank you very much, I'll close this issue then :)
